source/install_scripted_offline_rhel7_rhel8.rst

```diff
@@ -1,103 +1,84 @@
-Scripted Installation
+Offline Scripted Installation (RHEL 7/8 and CentOS 7/8)
 ----------------------
 The scripted installation runs an installer that uses Ansible to install the command line tool, quipucords server image, and the database image. When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can change some of the defaults used by the scripted installation with the `Installation Options <install.html#install-opts>`_.
 
 Obtaining the Installer
 ^^^^^^^^^^^^^^^^^^^^^^^
-1. Download the installer by entering the following command::
+1. Download & Install the quipucords installer rpm.
 
-    # curl -k -O -sSL https://github.com/quipucords/quipucords-installer/releases/latest/download/quipucords_install.tar.gz
+- RHEL & CentOS 7::
 
-2. Extract the installer by entering the following command::
+    # yum install -y https://github.com/quipucords/qpc-tools/releases/latests/download/quipucords_installer.el7.noarch.rpm
```

Comment:
I think this will now be the `qpc_tools.el7.noarch.rpm`

Reply:
yep